lib/tic_tac_toe_rules.rb

```diff
@@ -0,0 +1,40 @@
+class TicTacToeRules
+    def winner?(player_stats)
+        return false if player_stats.length <= 2 
+        evaluate_player_stats(player_stats)
+    end
+
+    def evaluate_player_stats(player_stats)
+        player_stats = player_stats.sort()
+        match_box = []
+
+        winning_combos.map do |combo|
+            match_box = []
+            player_stats.map do |stat|
+                if combo.include?(stat)
+                    match_box << stat
+                end
+            end
+            return true if match_box.length == 3
+        end
+        false
+    end
+
+    def winning_combos
```

Comment:
Should this method be public or private? 🤔  If yes/no, why?

Also, these combinations could be made more explicit as to which ones belong to what group of combinations by assigning them names. For example: `left_row_combo`.

Reply:
Yup. Also, because nothing else uses winning_combos. And it's likely nothing else will ever use it in the future. Except for this class.